common/lib/xmodule/xmodule/js/spec/video/video_events_plugin_spec.js

```diff
@@ -78,6 +78,13 @@
                 new_time: 1,
                 type: 'any'
             });
+            expect(state.videoEventsPlugin.emitPlayVideoEvent).toBeTruthy();
+        });
+
+        it('can emit "play_video" event after "seek_video" event ', function() {
+            state.videoEventsPlugin.emitPlayVideoEvent = false;
```

Comment:
why do we need this here? i.e. `state.videoEventsPlugin.emitPlayVideoEvent = false;`